modules/assert.js

```diff
@@ -248,6 +248,7 @@ function parseCode(code, offset) {
 }
 
 function getErrMessage(message, fn) {
+  return "assert.getErrMessage unsupported";

```

Comment:
What support is missing for this function?

Reply:
The error message format from quickjs is a little different from v8, this function can't get the correct result.